src/main/kotlin/com/lykke/trade/volume/monitoring/service/web/dto/MonitoringStatsDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.lykke.trade.volume.monitoring.service.web.dto
+
+import io.swagger.annotations.ApiModel
+
+@ApiModel("MonitoringStats")
+class MonitoringStatsDto(@JvmField val VmCpuLoad: Double,
```

Comment:
suggest using standard java naming convention, if we need capital letter in json, we can configure json serialization code to use starting capital letter

Reply:
fixed